src/VisualStudio/IntegrationTest/TestUtilities/Common/ProjectUtilities.cs

```diff
@@ -9,10 +9,13 @@ public abstract class Identity
 
     public class Project : Identity
     {
-        public Project(string name)
+        public Project(string name, string relativePath = null)
         {
             Name = name;
+            RelativePath = relativePath;
         }
+
+        public string RelativePath { get; }
```

Comment:
@basoundr Right; it seems like we should default `RelativePath` to <ProjectName>.<csproj\vbproj> if it isn't specified. Are there reasons that wouldn't work>